src/server/server.py

```diff
@@ -3955,6 +4018,17 @@ def heartbeat_thread_function():
             if (heartbeat_thread_function.iter_tracker % (4*HEARTBEAT_DATA_RATE_FACTOR)) == (2*HEARTBEAT_DATA_RATE_FACTOR):
                 CLUSTER.emitStatus()
 
+            # collect vrx lock status
+            if (heartbeat_thread_function.iter_tracker % (10*HEARTBEAT_DATA_RATE_FACTOR)) == 0:
+                if vrx_controller:
+                    # if vrx_controller.has_connection
+                    vrx_controller.get_node_lock_status()
+                    vrx_controller.request_variable_status()
+
+            if (heartbeat_thread_function.iter_tracker % (10*HEARTBEAT_DATA_RATE_FACTOR)) == 4:
+                # emit display status with offset
+                emit_vrx_list()
```

Comment:
Should 'emit_vrx_list()' be called if 'vrx_controller' is False?  Should this entire block of new code be conditional on 'vrx_controller'==True?

Reply:
Actually, seeing that 'vrx_controller' references an object, it seems like it should be set to 'None' if not used, and the tests should be either "if vrx_controller:" or "if vrx_controller == None:"

Reply:
We set to False to indicate it was not able to be loaded properly. I'll change it to just be None because the desired result is the same.

Reply:
The reason I bring it up is that if you set it to 'False' then it may be seen as a flag that would have True/False values.  I'm still wondering about my first question above.

Reply:
I agree and am addressing all of these, I just haven't fully tested it yet.